library/socketserver.po

```diff
@@ -184,6 +266,14 @@ msgid ""
 "each child.  In this case, you can use a threading server, but you will "
 "probably have to use locks to protect the integrity of the shared data."
 msgstr ""
+"Bien entendu, vous devrez toujours utiliser votre tête ! Par exemple, "
+"utiliser un serveur utilisant des processus enfants (*forking*) n'aurait "
```

Comment:
```suggestion
"utiliser un serveur utilisant des processus clonés (*forking*) n'aurait "
```

Si j'ai bien compris l'avertissement sur le fait qu'il fallait `os.fork`, c'est obligatoirement du multiprocessing fait par cette méthode et pas par par [l'une des autres](https://docs.python.org/3.10/library/multiprocessing.html#contexts-and-start-methods).

Reply:
Hum, je ne suis pas sur que l'auteur mette l'emphase là-dessus même si je pense que ce soit la méthode la plus naturelle puisque le processus fils continue l’exécution des différentes méthodes de la classe serveur ou du gestionnaire de requête comme si de rien n'était.

L'avertissement est plutôt que si la classe serveur contient des états modifiables par les requêtes reçues, il n'est pas possible d'utiliser le _multiprocessing_ en général car le clone de l'objet serveur n'a aucun moyen de remonter les changements d'état au parent.
C'est pour cela que l'auteur recommande d'utiliser des fils d’exécution dans ce cas.